### PR TITLE
Splitreus62

### PR DESCRIPTION
**Pull request by Na-Cly**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/splitreus62/splitreus62.c [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    // DDRD |= (1<<5);
+    // PORTD &= ~(1<<5);
+
+    // // orange led on
+    // DDRB |= (1<<0);
+    // PORTB &= ~(1<<0);
+
+	matrix_init_user();
+};
+
```
Comment (zvecr) [inline keyboards/splitreus62/splitreus62.c]:
```suggestion
```

File: keyboards/splitreus62/splitreus62.h [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+#ifdef USE_I2C
+#include <stddef.h>
+#ifdef __AVR__
+    #include <avr/io.h>
+    #include <avr/interrupt.h>
+#endif
+#endif
+
+
```
Comment (zvecr) [inline keyboards/splitreus62/splitreus62.h]:
```suggestion
```

File: keyboards/splitreus62/rules.mk [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+RGBLIGHT_ENABLE = yes       # Enable WS2812 RGB underlight.
+
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no    # Breathing sleep LED during USB suspend
+
+SPLIT_KEYBOARD = yes
+
+DEFAULT_FOLDER = splitreus62
```
Comment (zvecr) [inline keyboards/splitreus62/rules.mk]:
```suggestion
```

File: keyboards/splitreus62/readme.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+- https://github.com/Na-Cly/splitreus62
+- [Atreus](https://github.com/technomancy/atreus) Phil Hagelberg
+- [Atreus62](https://github.com/profet23/atreus62) Profet23
+
+A Split version of the Atreus62. You can copy keymaps from Atreus62 and use them with this keyboard.
+
+* Keyboard Maintainer: [NaCly](https://github.com/Na-Cly)
+* Hardware Supported: The PCBs, controllers supported
+* Hardware Availability: https://github.com/Na-Cly/splitreus62 
+### Make example for this keyboard (after setting up your build environment):
```
Comment (fauxpark) [inline keyboards/splitreus62/readme.md]:
```suggestion

Make example for this keyboard (after setting up your build environment):
```

File: keyboards/splitreus62/rules.mk [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = caterina
+
+# Build Options
+#   change to "no" to disable the options, or define them in the Makefile in
+#   the appropriate keymap folder that will get included automatically
+#
```
Comment (fauxpark) [inline keyboards/splitreus62/rules.mk]:
```suggestion
#   change yes to no to disable
#
```

**Comment by Na-Cly** [comment]
Let me know if you want anymore changes done. Thanks guys!

**Comment by Na-Cly** [comment]
I'll take a look at these. I do have a few extra PCBs. This isn't a hand wired project. I am not sure if I will stock the PCBs or not. 

Still should go into handwired?

**Comment by Na-Cly** [comment]
I'm not very good at Github. Did I need to change anything else?

I wasn't sure about the handwired thing since I may start producing these, and I do still have a couple to offload. 

Let me know.

Thanks for the help guys.

**Comment by zvecr** [comment]
`handwired` is kind of a misnomer in this case, given the hardware is currently open, I would say its okay for it where it is (but will defer to others).

**Comment by Na-Cly** [comment]
> `handwired` is kind of a misnomer in this case, given the hardware is currently open, I would say its okay for it where it is (but will defer to others).

That was my thinking on this too. 

Hey guys, can I get this merged soon? I have a few people who would love for this to be published so it can be used with the qmk configurator. 

Thanks!

File: keyboards/splitreus62/keymaps/default/keymap.c [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+        KC_LCTRL, KC_LGUI, KC_LALT, KC_NO, KC_NO, KC_NO, KC_NO,      KC_ENT, KC_SPC, KC_NO, KC_NO, KC_NO, KC_NO, TG(3)
+    ),
+        //game layer
+    [3] = LAYOUT(
+        KC_ESC, KC_1, KC_2, KC_3, KC_4, KC_5,                          KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINS,
+        KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T,                          KC_Y, KC_U, KC_I, KC_O, KC_P, KC_BSLS,
+        KC_LCTRL, KC_A, KC_S, KC_D, KC_F, KC_G,                        KC_H, LT(2, KC_J), KC_K, KC_L, KC_SCLN, KC_QUOT,
+        KC_LSFT, KC_Z, KC_X, KC_C, KC_V, KC_B,                         KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, KC_RSFT, 
+        KC_LCTRL, KC_NO, KC_LALT, KC_GRV, KC_SPC, KC_BSPC, KC_DEL,      KC_ENT, KC_SPC, KC_LBRC, KC_RBRC, KC_HOME, KC_END, TO(0)
+    )
```
Comment (noroadsleft) [inline keyboards/splitreus62/keymaps/default/keymap.c]: Just makes the keymap easier to read for when users of this board copy it. No logic changes.

```suggestion
    [0] = LAYOUT(
        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,       KC_5,                      KC_6,    KC_7,       KC_8,    KC_9,    KC_0,    KC_MINS,
        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,       KC_T,                      KC_Y,    KC_U,       KC_I,    KC_O,    KC_P,    KC_BSLS,
        KC_LCTL, KC_A,    KC_S,    KC_D,    LT(1,KC_F), KC_G,                      KC_H,    LT(2,KC_J), KC_K,    KC_L,    KC_SCLN, KC_QUOT,
        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,       KC_B,                      KC_N,    KC_M,       KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,
        KC_LCTL, KC_LGUI, KC_LALT, KC_GRV,  KC_EQL,     KC_BSPC, KC_DEL,  KC_ENT,  KC_SPC,  KC_LBRC,    KC_RBRC, KC_HOME, KC_END,  KC_ESC
    ),
    //Holding F
    [1] = LAYOUT(
        KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,                     KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,
        KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,                     KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,
        KC_LCTL, KC_NO,   KC_NO,   KC_NO,   _______, KC_NO,                     KC_NO,   KC_LEFT, KC_DOWN, KC_UP,   KC_RGHT, KC_NO,
        KC_LSFT, KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,                     KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_RSFT,
        KC_LCTL, KC_LGUI, KC_LALT, KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_HOME, KC_END,  KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO
    ),
    //Holding J
    [2] = LAYOUT(
        KC_MUTE, KC_VOLD, KC_VOLU, KC_NO,   KC_NO,   KC_NO,                     KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,
        KC_NO,   KC_MPRV, KC_MPLY, KC_MNXT, KC_NO,   KC_NO,                     KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,
        KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,                     KC_NO,   _______, KC_NO,   KC_NO,   KC_NO,   KC_NO,
        KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,                     KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,
        KC_LCTL, KC_LGUI, KC_LALT, KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_ENT,  KC_SPC,  KC_NO,   KC_NO,   KC_NO,   KC_NO,   TG(3)
    ),
    //game layer
    [3] = LAYOUT(
        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,                      KC_6,    KC_7,       KC_8,    KC_9,    KC_0,    KC_MINS,
        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,                      KC_Y,    KC_U,       KC_I,    KC_O,    KC_P,    KC_BSLS,
        KC_LCTL, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,                      KC_H,    LT(2,KC_J), KC_K,    KC_L,    KC_SCLN, KC_QUOT,
        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,                      KC_N,    KC_M,       KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,
        KC_LCTL, KC_NO,   KC_LALT, KC_GRV,  KC_SPC,  KC_BSPC, KC_DEL,  KC_ENT,  KC_SPC,  KC_LBRC,    KC_RBRC, KC_HOME, KC_END,  TO(0)
    )
```

File: keyboards/splitreus62/readme.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+# splitreus62
+
+- https://github.com/Na-Cly/splitreus62
+- [Atreus](https://github.com/technomancy/atreus) Phil Hagelberg
+- [Atreus62](https://github.com/profet23/atreus62) Profet23
+
+A Split version of the Atreus62. You can copy keymaps from Atreus62 and use them with this keyboard.
```
Comment (noroadsleft) [inline keyboards/splitreus62/readme.md]: I think this is a bit cleaner, but it retains all the information. Your link is under Hardware Availability, so it doesn't need to be in the preamble.

```suggestion
A split version of the [Atreus62](https://github.com/profet23/atreus62) by Profet23, which itself is based on Phil Hagelberg's [Atreus](https://github.com/technomancy/atreus). You can copy keymaps from Atreus62 and use them with this keyboard.
```

File: keyboards/splitreus62/readme.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+
+* Keyboard Maintainer: [NaCly](https://github.com/Na-Cly)
+* Hardware Supported: The PCBs, controllers supported
+* Hardware Availability: https://github.com/Na-Cly/splitreus62
```
Comment (noroadsleft) [inline keyboards/splitreus62/readme.md]:
```suggestion
* Hardware Supported: splitreus62 PCBs, Pro Micro
* Hardware Availability: https://github.com/Na-Cly/splitreus62
```

File: keyboards/splitreus62/rules.mk [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+NKRO_ENABLE = yes            # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
+MIDI_ENABLE = no            # MIDI controls
+AUDIO_ENABLE = no           # Audio output on port C6
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+RGBLIGHT_ENABLE = yes       # Enable WS2812 RGB underlight.
+
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no    # Breathing sleep LED during USB suspend
```
Comment (noroadsleft) [inline keyboards/splitreus62/rules.mk]: Aligns this section to QMK's AVR `rules.mk` template.

Also, you have full Bootmagic enabled, but I'd recommend using [Bootmagic Lite](https://docs.qmk.fm/#/feature_bootmagic?id=bootmagic-lite) instead, as it opens less opportunities for weirdness.

```suggestion
BOOTMAGIC_ENABLE = yes      # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = yes        # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
MIDI_ENABLE = no            # MIDI controls
BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
AUDIO_ENABLE = no           # Audio output on port C6
FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
UNICODE_ENABLE = no         # Unicode
```

File: keyboards/splitreus62/splitreus62.h [hunk 10]
```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT(\
+    L00, L01, L02, L03, L04, L05,                       R05, R04, R03, R02, R01, R00, \
+    L10, L11, L12, L13, L14, L15,                       R15, R14, R13, R12, R11, R10, \
+    L20, L21, L22, L23, L24, L25,                       R25, R24, R23, R22, R21, R20, \
+    L30, L31, L32, L33, L34, L35,                       R35, R34, R33, R32, R31, R30, \
+    L40, L41, L42, L43, L44, L45, L55,                  R55, R45, R44, R43, R42, R41, R40 \
```
Comment (noroadsleft) [inline keyboards/splitreus62/splitreus62.h]: Just aligns this section better visually.

```suggestion
    L40, L41, L42, L43, L44, L45, L55,             R55, R45, R44, R43, R42, R41, R40  \
```

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**